src/reanimated2/layoutReanimation/web/animation/Bounce.web.ts

```diff
@@ -0,0 +1,165 @@
+import { parseAnimationObjectToKeyframe } from '../animationParser';
+
+const DEFAULT_BOUNCE_TIME = 0.6;
+
+export const BounceInData = {
+  BounceIn: {
+    name: 'BounceIn',
+    style: {
+      0: { transform: [{ scale: 0 }] },
+      55: { transform: [{ scale: 1.2 }] },
+      70: { transform: [{ scale: 0.9 }] },
+      85: { transform: [{ scale: 1.1 }] },
+      100: { transform: [{ scale: 1 }] },
+    },
+    duration: DEFAULT_BOUNCE_TIME,
+  },
+
+  BounceInRight: {
+    name: 'BounceInRight',
+    style: {
+      0: { transform: [{ translateX: '100vw' }] },
+      55: { transform: [{ translateX: '-20px' }] },
+      70: { transform: [{ translateX: '10px' }] },
+      85: { transform: [{ translateX: '-10px' }] },
+      100: { transform: [{ translateX: '0px' }] },
+    },
+    duration: DEFAULT_BOUNCE_TIME,
+  },
```

Comment:
Sometimes there's an additional newline after each animation and sometimes there's no newline, let's unify it.

Reply:
Done in [5aafc4d](https://github.com/software-mansion/react-native-reanimated/pull/4805/commits/5aafc4d8a2c09e39ccdf76d490e7446b27320ae2).